packages/devtools_app/test/test_infra/utils/deep_links_utils.dart

```diff
@@ -13,6 +14,9 @@ class DeepLinksTestController extends DeepLinksController {
     return null;
   }
 
+  @override
+  bool get enableIosCheck => true;
+
```

Comment:
instead of adding this extra getter, you can just set `FeatureFlags.deepLinkIosCheck = true` in the test. Or in the constructor of DeepLinksTestController too if you want